blog/_posts/2020-10-02-false-positives.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Google warns about root.cern"
+layout: archive
+author: Axel Naumann
+---
+
+Thank you for your reports of Google warning about some pages on https://root.cern. Let me give you some background:
+
+We have investigated virus scanner reports on some of ROOT's binaries. The ROOT team has invested about 2.5 work days into this investigation: we took this seriously. On top of that, CERN IT's security team has investigated this independently, and also invested a notable amount of working hours. We were able to create binaries that were diagnosed as infected, by building `hadd.exe` "from scratch". We are convinced that these reports are false positives. We found out that upgrading the compiler works around the issue. 
```

Comment:
```suggestion
We have investigated virus scanner reports on some of ROOT's binaries. The ROOT team has invested the last working days since the incident trying to sort out this issue: we took this seriously. On top of that, CERN IT's security team has investigated this independently, and also invested a notable amount of working hours. We were able to create binaries that were diagnosed as infected, by building `hadd.exe` "from scratch". We are convinced that these reports are false positives. We found out that upgrading the compiler works around the issue. 
```
I personally wouldn't write a fixed amount of working days